server/docker-compose.yml

```diff
@@ -1,19 +1,35 @@
 version: "3.4"
 services:
+  proxy:
+    platform: linux/amd64/v3
+    image: nginx:latest
+    depends_on:
+      - "alfred"
+      - "historian"
+      - "nexus"
+    volumes:
+      - ./routerlicious/nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - "3003:3003"
+      - "3002:3002"
+      - "3001:3001"
   alfred:
     image: ${REGISTRY_URL:-mcr.microsoft.com}/fluidframework/routerlicious/server:${ALFRED_IMAGE_TAG:-latest}
-    ports:
-      - "3003:3000"
+    expose:
+      - "3000"
     command: node packages/routerlicious/dist/alfred/www.js
     environment:
       - DEBUG=fluid:*
       - NODE_ENV=development
       - IS_FLUID_SERVER=true
     restart: always
+    # THIS VOLUMES IS FOR LOCAL TESTING, IT SHOULD NOT GET MERGED
+    volumes:
+      - ./routerlicious/packages/routerlicious/config/config.json:/usr/src/server/packages/routerlicious/config/config.json
```

Comment:
Used this locally to validate running e2e tests against the latest server image, but with whole summary upload enabled (the image itself doesn't do it). This will go away before merging.